extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeStreamExtractorAgeRestrictedTest.java

```diff
@@ -71,10 +71,9 @@ public void testGetUploaderName() throws ParsingException {
         assertFalse(extractor.getUploaderName().isEmpty());
     }
 
-    @Ignore // Currently there is no way get the length from restricted videos
     @Test
     public void testGetLength() throws ParsingException {
-        assertTrue(extractor.getLength() > 0);
+        assertEquals(1789, extractor.getLength());
     }
 
     @Test
```

Comment:
Remove `@Ignore` from `testGetAudioStreams()`, now it seems to work.

Reply:
All right. However now subtitle tests seem to fail on my end.